File: hw/nase/u2/hw09StoreApp/JavaScript/store.js
//Jennyfer Nase
const Product = require('./product');
const Tax = require('./tax');


let id = 1;
let description = "computer";
let price = 100;

let pvp = Tax.computeTotal(price, 15);

let product = new Product(id, description, price);

console.log("product --> " + product.toString());


id = 2;
description = "mouse";
price = 1000;

pvp = Tax.computeTotal(price, 15);

let product2 = new Product(id, description, price, pvp);

console.log("product 2 --> " + product2.toString());

File: hw/nase/u2/hw09StoreApp/Python/store.py
from product import Product
from tax import Tax
def main():

    # Producto 1
    id = 1
    description = "computer"
    price = 100

    pvp = Tax.compute_total(price, 15)

    product = Product(id, description, price)

    print("product -->", product)

    # Producto 2
    id = 2
    description = "mouse"
    price = 1000

    pvp = Tax.compute_total(price, 15)

    product2 = Product(id, description, price, pvp)

    print("product 2 -->", product2)


if __name__ == "__main__":
    main()